dist/s_clang-format

```diff
@@ -1,24 +1,39 @@
-#! /bin/sh
+#! /bin/bash
 
-# Installation of the clang development package isn't standard, list a
-# couple of the places we're using.
-export PATH=$PATH:/usr/local/clang60/bin:/usr/local/llvm-devel/bin
+set -o pipefail
+
+download_clang_format() {
+	echo "$0: downloading clang-format for $(uname)"
+	if [ `uname` = "Linux" ]; then
+		wget https://s3.amazonaws.com/boxes.10gen.com/build/clang-format-3.8-rhel55.tar.gz -O dist/clang-format.tar.gz
+		tar --strip=2 -C dist/ -xf dist/clang-format.tar.gz build/bin/clang-format && rm dist/clang-format.tar.gz
+	elif [ `uname` = "Darwin" ]; then
+		wget https://s3.amazonaws.com/boxes.10gen.com/build/clang%2Bllvm-3.8.0-x86_64-apple-darwin.tar.xz -O dist/clang-format.tar.gz
+		tar --strip=2 -C dist/ -xf dist/clang-format.tar.gz clang+llvm-3.8.0-x86_64-apple-darwin/bin/clang-format && rm dist/clang-format.tar.gz
+	else
+		echo "$0: unsupported environment"
```

Comment:
I'd suggest dumping the environment in the error.

Reply:
a9ddfa2